docs/Model_Assumptions.rst

```diff
@@ -176,12 +176,117 @@ The confersion factors are derived from their `Gasoline Gallon Equivalents.<http
 Limitations
 -----------
 
+When running simulations with the MVS, there are certain peculiarities to be aware of. The peculiarities can be considered as limitations, some of which are merely assumptions and others drawbacks of the model. Some of those are inherited due to the nature of the MVS and its underlying modules, and some can still be addressed in the future during the MVS development process, which is still ongoing. The following table lists the MVS limitations based on their type.
+
+.. list-table:: Limitations
+   :widths: 25 25 25
+   :header-rows: 1
+
+   * - Type
+     - Limitation
+     - Link
+   * - Inherited
+     - Disregard of some real life constraint
+     - :ref:`disregard`
+   * - Inherited
+     - Simplified asset model
+     - 
+   * - Inherited
+     - No degradation
+     - 
+   * - Inherited
+     - Perfect foresight
+     - 
+   * - Can be addressed
+     - Extension of KPIs
+     - 
+   * - Can be addressed
+     - Random excess energy distribution
+     - 
+   * - Can be addressed
+     - Renewable energy share defintion
+     - 
+   * - Can be addressed
+     - Energy carrier rating
+     -      
+   * - Can be addressed
+     - Energy shortage or grid interruption
+     - 
+   * - Can be addressed
+     - Generic cost definitions
+     -  
+   * - Can be addressed
+     - Unlinked one technical unit transformers
+     -      
+
+.. _disregard:
+Disregard of some real life constraint
+######################################
+:Limitation: Some real life constraint are not taken into consideration in the MVS.
+:Reason: The MVS is based oemof-solph python library and uses its generic components to set up an energy system, which allow unidirectional flows only. 
+:Implications: This means that the MVS might result in unfeasible dispatch of assets. For instance, the battery might be charged by the rectifier and discharged by the inverter at the same time step t. Another case that could occur is feeding the grid and consuming from it at the same time t. If those constraints were to be implemented, the optimization problem would become a non-linear one.
```

Comment:
```suggestion
:Implications: This limitation means that the MVS might result in unfeasible dispatch of assets. For instance, the a bus might supplied by a rectifier and itself supply an inverter at the same time step t, which can logically not be if these assets are part of one physical bi-directional inverter. Another case that could occur is feeding the grid and consuming from it at the same time t.
```

Actually, a battery is a special case: Here we actually CAN do something to prevent simultaneous charge and discharge: We can define a minimal `dispatch_cost` for charge or discharge, and thus both should not take place in parallel (except if the flow must go though the battery to reach another bus). 

You can also mention this here, if it fits above narrative.